connections/src/main/java/com/stripe/android/connections/ConnectionsSheetActivity.kt

```diff
@@ -4,6 +4,9 @@ import android.app.Activity
 import android.content.Intent
 import android.net.Uri
 import android.os.Bundle
+import android.util.Log
```

Comment:
Unused import, you could enable detekt on CI

Reply:
It's enabled, Github Actions are down 😢  https://twitter.com/githubstatus/status/1504102675515908099?cxt=HHwWhoC5_cHa0t8pAAAA